pipeline/metadata/test_schema.py

```diff
@@ -159,6 +159,12 @@ def test_satellite_flatten_matches_bq_schema(self) -> None:
               tls_version = 123,
               tls_cipher_suite = 3,
               tls_cert = 'MII...',
+              tls_cert_common_name = None,
+              tls_cert_issuer = None,
+              tls_cert_start_date = None,
+              tls_cert_end_date = None,
+              tls_cert_alternative_names = [],
+              matched_cert = None,
```

Comment:
@avirkud could you point me to the place where the `matched_cert` is generated?

Reply:
Removed the field since it's not calculated in this PR